pkg/virtual/workspaces/registry/rest.go

```diff
@@ -203,12 +192,22 @@ func withoutGroupsWhenPersonal(user user.Info, scope string) user.Info {
 	return user
 }
 
+func (s *REST) extractOrg(ctx context.Context) (orgName string, org Org, err error) {
+	orgName = ctx.Value(WorkspacesOrgKey).(string)
```

Comment:
am not a fan to introduce another mechanism here to get a cluster name into a context. We have `genericapirequest.ClusterNameFrom(ctx)` and a corresponding setter. Can we use those?

Reply:
Well, the `WorkspacesOrgKey` was added by you as a key distinct from the `Cluster` key in the `RootPathResolver`.
Additionally, it the `cluster` context value is already set [here](https://github.com/kcp-dev/kcp/blob/main/pkg/virtual/framework/rootapiserver/root_apiserver.go#L171) to please the discovery for all `fixedgvs` virtua workspaces.
I'd rather manage "organization" and "logical cluster" as 2 distinct context values, even though an organization is finally implemented as a logical cluster.

Reply:
Renamed in commit c48becc80ef541475a0e032a26cfa6a9024c5ac8